src/saving/SaveHelper.cs

```diff
@@ -22,9 +22,52 @@ public enum SaveOrder
         FileSystem,
     }
 
+    public static void RemoveExcessAutoSaves()
```

Comment:
It seems like this method isn't called from anywhere.
I think a method similar to `QueueRemoveExcessQuickSaves()` needs to be created and hooked up.